package.json

```diff
@@ -2,30 +2,39 @@
   "name": "@producktivity/source",
   "license": "MIT",
   "private": true,
-  "type": "module",
+  "type": "commonjs",
```

Comment:
we use script in module type for make it more capable in ESM pls change back to `module`